syntax = "proto3";

package Wwbweibo.CrackDetect.Models;

message TaskItemModel{
	// 任务ID
	string majorTaskId=1;
	// 子任务ID
	string subTaskId=3;
	// 子任务时间
	string subTaskTime=5;
	// 地理位置
	.Wwbweibo.CrackDetect.Models.Position position=6;
}

message Position{
	double longitude=1;
	double latitude=2;
}